src/main/java/org/opensearch/security/configuration/DlsQueryParser.java

```diff
@@ -15,151 +15,151 @@
 
 package org.opensearch.security.configuration;
 
-import java.io.IOException;
 import java.util.Set;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
The explicit `log4j` dependencies should be removed. Please reference to this commit: https://github.com/opensearch-project/security/commit/12ae0cba4e5bdbe45e85f35078cc785c99cf9ab6